scripts/ci/check_compliance.py

```diff
@@ -369,7 +369,7 @@ class KconfigCheck(ComplianceTest):
     """
     name = "Kconfig"
     doc = "See https://docs.zephyrproject.org/latest/build/kconfig/tips.html for more details."
-    path_hint = "<zephyr-base>"
+    path_hint = "<git-top>"
```

Comment:
Same here, the check is actually performed on `ZEPHYR_BASE`, changing this feels iffy.

Reply:
When re-using compliance checks from the main repository, and running compliance checks in the module root, this is not iffy. It is necessary.